packages/myst-cli/src/build/site/template.ts

```diff
@@ -31,6 +34,7 @@ export async function getSiteTemplate(session: ISession, opts?: { defaultTemplat
         ruleId: RuleId.validSiteConfig,
       });
     },
+    validateFiles: false,
```

Comment:
Can we only turn this on if the template is specified?

```suggestion
    validateFiles: opts?.template ? false : true,
```